src/interpreter/mod.rs

```diff
@@ -1414,14 +1408,22 @@ pub fn interpret_start_points<'a, 'tcx>(
 
                 gecx.push_stack_frame(tcx.map.local_def_id(id), mir.span, CachedMir::Ref(mir), substs, return_ptr);
 
-                match (gecx.run(), return_ptr) {
-                    (Ok(()), Some(ptr)) => if log_enabled!(::log::LogLevel::Debug) {
+                loop { match (stepper::step(&mut gecx), return_ptr) {
```

Comment:
done

On Fri, 10 Jun 2016 07:30:36 -0700
Scott Olson <notifications@github.com> wrote:

> > @@ -1414,14 +1408,22 @@ pub fn interpret_start_points<'a, 'tcx>(
> >  
> >                  gecx.push_stack_frame(tcx.map.local_def_id(id),
> > mir.span, CachedMir::Ref(mir), substs, return_ptr); 
> > -                match (gecx.run(), return_ptr) {
> > -                    (Ok(()), Some(ptr)) => if
> > log_enabled!(::log::LogLevel::Debug) {
> > +                loop { match (stepper::step(&mut gecx),
> > return_ptr) {
> 
> Nit: split this up into two lines
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/solson/miri/pull/20/files/8c3a066d8d6fec4720c171130816c97ed209ce95..3b804942fdcc8858644218a570e034062021c7a4#r66622654